pkg/kv/kvserver/logstore/sideload_disk.go

```diff
@@ -217,7 +215,7 @@ func (ss *DiskSideloadStorage) TruncateTo(
 			err = nil // handled
```

Comment:
Wonder if the assignment above should be `err = Remove()` rather than `err := Remove()`. Otherwise, this `err = nil` seems like a no-op.

Reply:
It might serve to confirm to the reader (or a linter) that we're intentionally falling through and this isn't a case of a forgotten `return err`. I agree that the nil assignment does not change the program.

Reply:
The code only ignores `IsNotExist` errors. Looks good to me, leaving as is.